packages/mobile/src/components/audio/RNVideoAudioPlayer.tsx

```diff
@@ -64,6 +64,8 @@ const { getIsReachable } = reachabilitySelectors
 
 const { getNftAccessSignatureMap } = gatedContentSelectors
 
+// const bufferTimes: number[] = []
```

Comment:
sup with this?

Reply:
Just some test data logic I was using. I can delete it but I feel like we're probably about to delete this whole component, so I didnt bother (This is the RN Video component)